frontend/test/ui_utils.div.test.tsx

```diff
@@ -1,60 +1,72 @@
 import * as React from 'react'
 import {div, css} from '../src/ui_utils'
 import {render, fireEvent, screen} from '@testing-library/react'
+// import serializer from 'jest-emotion'
+import renderer from 'react-test-renderer'
+// import styled from '@emotion/styled'
```

Comment:
I'm not a big fan of keeping commented code as you know - but it's not like it does any harm...

Reply:
i'll remove these